rpc/types_transaction.go

```diff
@@ -327,7 +357,7 @@ type DeployTransactionProperties struct {
 
 type BroadcastedDeployTxn struct {
```

Comment:
This change will cause merge conflicts when you address [this](https://github.com/NethermindEth/starknet.go/pull/286#discussion_r1311156134) and merge #286 .
Let's merge that one first and then fix this one